tasks-by-users/Rehan-Khan-Engineer/06-Loading-Animation-CSS/style.css

```diff
@@ -0,0 +1,82 @@
+body {
+  margin: 0;
+  padding: 0;
+  background: #262626;
+  /* height: 100vh;
+    display: flex;
+    align-items: center;
+    justify-content: center; */
+}
+
+.jj-ring {
+  position: absolute;
+  top: 50%;
+  left: 50%;
+  transform: translate(-50%, -50%);
+  height: 150px;
+  width: 150px;
+  border: 3px solid #3c3c3c;
+  background: transparent;
+  text-align: center;
+  text-transform: uppercase;
+  line-height: 150px;
+  border-radius: 50%;
+  color: yellow;
```

Comment:
Group Similar Properties so it will look more clear @Rehan-Khan-Engineer

Reply:
Done @Abbas4tech